README.md

```diff
@@ -1,5 +1,11 @@
 # Open Operations Manifesto
 
+## Licenses
+
+The content of this repository is published under multiple licenses.
+The manifesto itself is licensed under [CC-BY-SA 4.0](LICENSE-DOCS).
```

Comment:
I in general like CC-BY-SA-4.0, a weak copyleft license for non-code (and not-compiled code).
For this specific manifesto, I would be fine to drop the SA (share-alike) clause. In other words: If someone wants to copy content from the manifesto and then use it under a restrictive license (internally?) I would not mind. I would keep the attribution clause (BY) though, so credit goes to where credit is due.
It's probably not very relevant -- the manifesto gets its power by being shared, so anyone wanting to create a derived manifesto is likely to publish it to have any effect ...

Reply:
Same here. Let's ditch the `Share-Alike` feature for the sake of a possible higher adoption rate.